hack/generate-admin-kubeconfig.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+hostname=$(cat ./hack/kcp/values.yaml | grep externalHostname | cut -d" " -f2- | tr -d '"')
```

Comment:
perhaps it might be cleaner to do `--set externalHostname=$EXTERNAL_HOSTNAME` in kind-setup.sh, then export that variable for use here?

Or else this could be inlined in the script, if we don't expect it to be called outside the script?